bentoml/_internal/bento/docker/templates/base.j2

```diff
@@ -44,6 +44,7 @@ ENV {{ key }}={{ value }}
 ARG BENTO_PATH={{ bento__path }}
 ENV BENTO_PATH=$BENTO_PATH
 ENV BENTOML_HOME={{ bento__home }}
+ENV HOME={{ bento__home }}
```

Comment:
I don't think this would work either, since this will mess up with install other library that use $HOME.

Currently $HOME is setting to /root, not /home/bentoml